spec/operations/workflows/unretire_subjects_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Workflows::UnretireSubjects do
+  let(:api_user) { ApiUser.new(build_stubbed(:user)) }
+  let(:workflow) { create(:workflow) }
+  let(:subject_set) { create(:subject_set, project: workflow.project, workflows: [workflow]) }
+  let(:subject_set_id) { subject_set.id }
+  let(:subject1) { create(:subject, subject_sets: [subject_set]) }
+  let(:subject2) { create(:subject, subject_sets: [subject_set]) }
+  let(:subject_workflow_count) { create }
```

Comment:
i think this can be deleted - I don't think this creates a SubjectWorkflowStatus (note the model name differs from the table name for historical reasons) and isn't used..